src/app/login/login.component.html

```diff
@@ -1,29 +1,46 @@
-            <p class="control">
-            <input type="password" class="input" name="password" [(ngModel)]="model.password" #password="ngModel" placeholder="password" required />
-            </p>
-            <div *ngIf="f.submitted && !password.valid" class="help is-danger level-left">Password is required</div>
-        </div>
+<section class="is-success">
+    <ng-progress [positionUsing]="'marginLeft'" [minimum]="0.15" [maximum]="1" [speed]="200" [showSpinner]="true" [direction]="'leftToRightIncreased'"
+        [color]="'#1B95E0'" [trickleSpeed]="300" [thick]="true" [ease]="'linear'">
+    </ng-progress>
+    <div class="has-text-centered">
+        <div class="column is-4 is-offset-4">
+            <div class="box">
+                <form name="form" (ngSubmit)="f.form.valid && login()" #f="ngForm" novalidate>
+                    <div class="field">
+                        <div class="control has-icons-left">
+                            <input type="text" class="input" name="username" [(ngModel)]="model.username" #username="ngModel" placeholder="User Name"
+                                required />
+                            <span class="icon is-small is-left">
+                                <i class="fa fa-user"></i>
+                            </span>
+                        </div>
+                        <div *ngIf="f.submitted && !username.valid" class="help is-danger level-left">Username is required</div>
+                    </div>
 
-        <div class="field">
-            <p class="control">
-                <button  class="button is-primary">Login</button>
+                    <div class="field">
+                        <div class="control has-icons-left">
+                            <input type="password" class="input" name="password" [(ngModel)]="model.password" #password="ngModel" placeholder="Password"
+                                required />
+                            <span class="icon is-small is-left">
+                                <i class="fa fa-lock"></i>
+                            </span>
+                        </div>
+                        <div *ngIf="f.submitted && !password.valid" class="help is-danger level-left">Password is required</div>
+                    </div>
+                    <div class="field is-grouped">
+                        <div class="control">
+                            <button type="button" class="button is-small is-text" (click)="skip()">skip</button>
+                        </div>
+                        <div class="control">
+                            <button class="button is-small is-info">Login</button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <p class="has-text-grey">
+                <a (click)="setupInstance()">Setup Instance</a> &nbsp;.&nbsp;
+                <a href="#">Forgot Password?</a> &nbsp;&nbsp;
```

Comment:
please remove this Forgot password stuff; or add alert on this <Ask the administrator to set a password>